### ISSUE TITLE
object measurement introduce eccentricity in explanation, python circularity calculation

### ISSUE THREAD

**Issue by manerotoni** [NEUBIAS/training-resources#690]
in the python code we show eccentricity and it would be nice to get an idea of what it is. 
Add some drawing of the different metrics for helping to illustrate shape measurements. 

Add code for circularity calculation

**Comment by tischi** [comment]
Yes great! Either to main figure or to another figure in the "Explanations" section.

**Comment by tischi** [comment]
Related to https://github.com/NEUBIAS/training-resources/issues/666
